app/views/pets/show.html.erb

```diff
@@ -5,7 +5,11 @@
 <p>Sex: <%= @pet.sex %></p>
 <p>Adoption Status: <%= @pet.adoption_status %></p>
 
+<% if already_favorited(@pet) %>
+  <p> This pet is already favorited </p>
```

Comment:
Isn't this supposed to be a flash notice?

Reply:
Yeah, I was taking it one bite at a time. That was a placeholder until I could implement the delete link in it's place.

Reply:
I read more on some of the responses folks gave me when I posted last night.  This seems to be the route some folks took.